modulemd/v2/include/modulemd-2.0/modulemd-module-stream-v1.h

```diff
@@ -756,6 +842,32 @@ modulemd_module_stream_v1_remove_runtime_requirement (
   ModulemdModuleStreamV1 *self, const gchar *module_name);
 
 
+/**
+ * modulemd_module_stream_v1_clear_buildtime_requirement
+ * @self: (in): This #ModulemdModuleStreamV1 object.
+ *
+ * Remove all buildtime dependencies for this module.
+ *
+ * Since: 2.5
+ */
+void
+modulemd_module_stream_v1_clear_buildtime_requirement (
```

Comment:
Again, this and the next function feel like they should be named in the plural form.

Reply:
This one was an oversight. I'll fix that right up.

Reply:
Fixed and force-pushed.